src/Compilers/CSharp/Portable/Binder/PatternSwitchBinder2.cs

```diff
@@ -97,45 +123,28 @@ internal override void BindPatternSwitchLabelForInference(CasePatternSwitchLabel
         /// </summary>
         private ImmutableArray<BoundPatternSwitchSection> BindPatternSwitchSections(
             Binder originalBinder,
-            out BoundPatternSwitchLabel defaultLabel,
-            out bool isComplete,
-            out bool someCaseMatches,
-            DiagnosticBag diagnostics)
+            DiagnosticBag diagnostics,
+            out BoundPatternSwitchLabel defaultLabel)
         {
-            defaultLabel = null;
-
-            // someCaseMatches will be set to true if some single case label would handle all inputs
-            someCaseMatches = false;
-
             // Bind match sections
             var boundPatternSwitchSectionsBuilder = ArrayBuilder<BoundPatternSwitchSection>.GetInstance();
-            SubsumptionDiagnosticBuilder subsumption = new SubsumptionDiagnosticBuilder(ContainingMemberOrLambda, SwitchSyntax, this.Conversions, SwitchGoverningType);
+            defaultLabel = null;
             foreach (var sectionSyntax in SwitchSyntax.Sections)
             {
-                var section = BindPatternSwitchSection(sectionSyntax, originalBinder, ref defaultLabel, ref someCaseMatches, subsumption, diagnostics);
+                var section = BindPatternSwitchSection(sectionSyntax, originalBinder, ref defaultLabel, diagnostics);
                 boundPatternSwitchSectionsBuilder.Add(section);
             }
 
-            isComplete = defaultLabel != null || subsumption.IsComplete || someCaseMatches;
             return boundPatternSwitchSectionsBuilder.ToImmutableAndFree();
         }
 
         /// <summary>
         /// Bind the pattern switch section, producing subsumption diagnostics.
```

Comment:
>producing subsumption diagnostics [](start = 45, length = 33)

Is this still correct, is this method "producing subsumption diagnostics"?

Reply:
No, will fix. Addressed https://github.com/dotnet/roslyn/pull/24059/commits/114a56e9077ed03f08443889b69cae6972a4acc6 of https://github.com/dotnet/roslyn/pull/24059